### DISCUSSION TITLE
How i see if someone erase folders or files?

### DISCUSSION THREAD

**Discussion by Hadoc1987** [iterate-ch/cyberduck#14330] [Q&A]
Hi, i have a company and i use cyberduck to share files with others companies.

The problem is, files and folders sometimes get deleted, even the ones i share, and i don't know for what reason.

I want to know, there is a way to see who has been deleting the files?

Thanks for the help.

**Comment by AliveDevil** [comment]
Check the server log files, to see, which user deleted the files.
If there are no log files then you're probably out of luck.
